vividus-allure-adaptor/src/main/java/org/vividus/bdd/report/allure/AllureReportGenerator.java

```diff
@@ -111,6 +116,20 @@ private void generateReport()
         LOGGER.info("Allure report is successfully generated at: {}", reportDirectory.getAbsolutePath());
     }
 
+    private void writeCategoriesInfo() throws IOException
+    {
+        List<AllureCategory> categories = Arrays.asList(
+                new AllureCategory("Test defects", Collections.singletonList(Status.BROKEN)),
+                new AllureCategory("Product defects", Collections.singletonList(Status.FAILED)),
+                new AllureCategory("Known issue", Collections.singletonList(Status.UNKNOWN)));
+
+        try (BufferedWriter writer = Files.newBufferedWriter(resultsDirectory.toPath().resolve("categories.json")))
```

Comment:
this logic of writing the data to the file is very similar to the logic from `writeExecutorInfo` so I propose to move the common part to a new method and reuse it

Reply:
Then we don't need to create ObjectMapper obejctMapper field?

Reply:
you can create `ObjectMapper` field or variable, the key is to create it once and reuse (do not create it in the method invoked twice)